modules/plottinggl/src/processors/scatterplotmatrixprocessor.cpp

```diff
@@ -125,13 +121,7 @@ ScatterPlotMatrixProcessor::ScatterPlotMatrixProcessor()
         }
     });
 
-    correlectionTF_.get().clear();
-    correlectionTF_.get().add(0.0, vec4(1, 0, 0, 1));
-    correlectionTF_.get().add(0.5, vec4(1, 1, 1, 1));
-    correlectionTF_.get().add(1.0, vec4(0, 0, 1, 1));
-    correlectionTF_.setCurrentStateAsDefault();
-
-    for (auto font : font::getAvailableFonts()) {
+    for (auto& font : font::getAvailableFonts()) {
```

Comment:
the does font property not handle this? should be in the constructor preferably

Reply:
`FontProperty` is a composite that handles this internally. Here only a plain `OptionPropertyString` is used. Could consider making a specialization for font faces.

Reply:
Added new `FontFaceOptionProperty` as a custom type for the font face.